src/test/resources/bad_wdl_version.describe.json

```diff
@@ -0,0 +1,13 @@
+{
+  "valid":false,
+  "errors":["ERROR: Finished parsing without consuming all tokens.\n\nversion plaid\n^\n     "],
```

Comment:
is the extra whitespace here - and in malformed_file.describe.json - relevant?

Reply:
cleaned up